### PR TITLE
Feature/detekt setup

### PR DESCRIPTION
**Pull request by kotrasatheesh-dev**
initial setup of detekt and required files

### CODE REVIEW

File: app/src/main/java/com/rickyandmonty/ui/ui/theme/Color.kt
Comment (shreyansh-mm) [inline app/src/main/java/com/rickyandmonty/ui/ui/theme/Color.kt:1]: This should be moved to a different module, should not be in app module.

File: app/src/main/java/com/rickyandmonty/ui/ui/theme/Theme.kt
Comment (shreyansh-mm) [inline app/src/main/java/com/rickyandmonty/ui/ui/theme/Theme.kt:1]: This should be moved to a different module, should not be in app module.

File: app/src/main/java/com/rickyandmonty/ui/ui/theme/Theme.kt [hunk 1]
```diff
@@ -22,15 +21,16 @@ private val LightColorScheme = lightColorScheme(
...
+    /*
+     * Other default colors to override
+     * background = Color(0xFFFFFBFE),
+     * surface = Color(0xFFFFFBFE),
+     * onPrimary = Color.White,
+     * onSecondary = Color.White,
+     * onTertiary = Color.White,
+     * onBackground = Color(0xFF1C1B1F),
+     * onSurface = Color(0xFF1C1B1F),
+     */
```
Comment (shreyansh-mm) [inline app/src/main/java/com/rickyandmonty/ui/ui/theme/Theme.kt]: Remove the commented code.

**Review by shreyansh-mm — CHANGES_REQUESTED** [review]
We are missing github action for detekt.
This action needs to run on every PR automatically.

File: app/src/main/java/com/rickyandmonty/ui/ui/theme/Color.kt
Comment (kotrasatheesh-dev) [reply app/src/main/java/com/rickyandmonty/ui/ui/theme/Color.kt:1]: multi module setup not done yet once it is done i will add

File: app/src/main/java/com/rickyandmonty/ui/ui/theme/Theme.kt
Comment (kotrasatheesh-dev) [reply app/src/main/java/com/rickyandmonty/ui/ui/theme/Theme.kt:1]: multi module setup not done yet once it is done i will add

**Review by shreyansh-mm — APPROVED** [review]

**Pull request merged**